src/glb-healthcheck/GLBTable.go

```diff
@@ -69,17 +70,24 @@ type GLBTable struct {
 	Backends []*GLBBackend `json:"backends"`
 }
 
-type GLBTableConfig struct {
-	Tables []*GLBTable `json:"tables"`
+type GLBHealthGlobalConfig struct {
+	TimeoutMilliSec  time.Duration `json:"timeout_ms,omitempty"`
+	IntervalMilliSec time.Duration `json:"interval_ms,omitempty"`
+	Trigger          int           `json:"trigger,omitempty"`
 }
 
-func LoadGLBTableConfig(filename string) (*GLBTableConfig, error) {
+type GLBGlobalConfig struct {
+	HealthcheckGlobalCfg *GLBHealthGlobalConfig `json:"healthcheck_global_cfg"`
```

Comment:
`_cfg` might be a bit superfluous in a config file, perhaps even `healthcheck_defaults` would be a better name, and also matches up with the idea that tables and backends could eventually override the defaults.
```suggestion
	HealthcheckGlobalCfg *GLBHealthGlobalConfig `json:"healthcheck_defaults"`
```

Reply:
On second thoughts, "healthcheck_defaults" has potential to be a little confusing.
Each of the 3 params has a default value. Those are being overridden by this config-stanza and this stanza is optional.
How about just calling it "healthcheck" instead?

Reply:
> How about just calling it "healthcheck" instead?

That seems fine to me, I guess we can also move towards them looking the same (structure wise) at multiple levels that way.

Reply:
So perhaps then `healthchecks` since that's what it is under `backends`.

Reply:
Sure.